testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -20,13 +20,15 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
+# autopep8: off
```

Comment:
What is this comment about?

Reply:
It was from the autostyler but I removed it in a previous commit